kogito-kafka-quickstart-springboot/README.md

```diff
@@ -71,8 +129,21 @@ To make use of this application it is as simple as putting a message on `travell
 
 this will then trigger the successful processing of the traveller and put another message on `processedtravellers` topic.
 
+* To examine ProcessedTravellers topic and verify it was auto sent
+
+```
+kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers --from-beginning
```

Comment:
As it is done for Kafka & Zookeeper starts, use `bin/kafka-console-consumer.sh` in kafka folder